graphql/typeDefs.ts

```diff
@@ -143,6 +143,7 @@ export default gql`
     name: String!
     isAdmin: Boolean!
     cliToken: String
+    discordRefreshToken: String
```

Comment:
we shouldn't expose this to the front end.

Reply:
oh true, got carried away

Reply:
hmm I think I want it in both the AuthResponse (upon login) and userInfoController. is a separate prisma query for the refreshToken more appropriate?